tests/ImageSharp.Tests/Formats/Jpg/JpegColorConverterTests.cs

```diff
@@ -467,58 +470,60 @@ private static void ValidateCyyK(in JpegColorConverter.ComponentValues values, i
+            float g = v.Y / MaxColorChannelValue;
+            float b = v.Z / MaxColorChannelValue;
+            var expected = new Rgb(r, g, b);
 
             var actual = new Rgb(result.Component0[i], result.Component1[i], result.Component2[i]);
-            var expected = new Rgb(v.X, v.Y, v.Z);
 
-            Assert.Equal(expected, actual, ColorSpaceComparer);
+            bool equal = ColorSpaceComparer.Equals(expected, actual);
+            Assert.True(equal, $"Colors {expected} and {actual} are not equal at index {i}");
         }
 
-        private static void ValidateRgb(in JpegColorConverter.ComponentValues values, in JpegColorConverter.ComponentValues result, int i)
+        private static void ValidateRgb(in JpegColorConverterBase.ComponentValues values, in JpegColorConverterBase.ComponentValues result, int i)
         {
-            float r = values.Component0[i];
-            float g = values.Component1[i];
-            float b = values.Component2[i];
+            float r = values.Component0[i] / MaxColorChannelValue;
+            float g = values.Component1[i] / MaxColorChannelValue;
+            float b = values.Component2[i] / MaxColorChannelValue;
+            var expected = new Rgb(r, g, b);
 
             var actual = new Rgb(result.Component0[i], result.Component1[i], result.Component2[i]);
-            var expected = new Rgb(r / 255F, g / 255F, b / 255F);
 
-            Assert.Equal(expected, actual, ColorSpaceComparer);
+            bool equal = ColorSpaceComparer.Equals(expected, actual);
+            Assert.True(equal, $"Colors {expected} and {actual} are not equal at index {i}");
         }
 
-        private static void ValidateGrayScale(in JpegColorConverter.ComponentValues values, in JpegColorConverter.ComponentValues result, int i)
+        private static void ValidateGrayScale(in JpegColorConverterBase.ComponentValues values, in JpegColorConverterBase.ComponentValues result, int i)
         {
-            float y = values.Component0[i];
+            float y = values.Component0[i] / MaxColorChannelValue;
+            var expected = new Rgb(y, y, y);
+
             var actual = new Rgb(result.Component0[i], result.Component0[i], result.Component0[i]);
-            var expected = new Rgb(y / 255F, y / 255F, y / 255F);
 
-            Assert.Equal(expected, actual, ColorSpaceComparer);
+            bool equal = ColorSpaceComparer.Equals(expected, actual);
+            Assert.True(equal, $"Colors {expected} and {actual} are not equal at index {i}");
         }
 
-        private static void ValidateCmyk(in JpegColorConverter.ComponentValues values, in JpegColorConverter.ComponentValues result, int i)
+        private static void ValidateCmyk(in JpegColorConverterBase.ComponentValues values, in JpegColorConverterBase.ComponentValues result, int i)
         {
-            var v = new Vector4(0, 0, 0, 1F);
-            var scale = new Vector4(1 / 255F, 1 / 255F, 1 / 255F, 1F);
+            Vector4 v = Vector4.Zero;
```

Comment:
This `v` vector does not seem to be used and can be removed?

Reply:
I will fix all warning right after that nasty failing test, thanks!